DEVELOPING_OPAL/validate/src/test/scala/org/opalj/fpcf/PurityTests.scala

```diff
@@ -97,4 +102,29 @@ class PurityTests extends PropertiesTest {
         validateProperties(as, contextsWithAnnotations(as.project), Set("Purity"))
     }
 
-}
+    describe(
+        "the org.opalj.fpcf.analyses.L2PurityAnalysis is executed "+
+            "together with the L0FieldImmutabilityAnalysis and L2FieldAssignabilityAnalysis"
+    ) {
+
+            L2PurityAnalysis.setRater(Some(SystemOutLoggingAllExceptionRater))
+
+            val as = executeAnalyses(Set(
+                EagerL2PurityAnalysis,
+                LazyL0FieldImmutabilityAnalysis,
```

Comment:
Please use the same ordering of analyses as for the other test cases to make it easier to compare the configurations.